infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/NettyLogReplicationClientChannelAdapter.java

```diff
@@ -44,9 +43,9 @@ public void connectAsync() {
         executorService.submit(() -> {
             ClusterDescriptor remoteCluster = getRemoteClusterDescriptor();
             for (NodeDescriptor node : remoteCluster.getNodesDescriptors()) {
-                log.info("Create Netty Channel to remote {}", node.getEndpoint());
+                log.info("Create Netty Channel to remote node {}", node.getRealNodeId());
```

Comment:
Similar to GRPC, can you also print the endpoint? it's easier to debug, something like: {}@{}:{}", nodeId, nodeLocator.getHost(), nodeLocator.getPort()

Reply:
Done.